### PR TITLE
[PROT-3171] Configure request body filtering

### PR DESCRIPTION
**Pull request by knizhegorodov**

### CODE REVIEW

File: spec/lib/kiev/request_body_filter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          "/",
+          "REQUEST_METHOD" => "POST",
+          "CONTENT_TYPE" => "application/json",
+          :input => "{\"password\":12345}"
+        )
+      )
+    end
+
+    it "doesn't filter " do
+      initial = Kiev.config.disable_filter_for_log_levels
```
Comment (everlast240) [inline spec/lib/kiev/request_body_filter_spec.rb:20]: around hook?

File: spec/lib/kiev/request_body_filter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          "/",
+          "REQUEST_METHOD" => "POST",
+          "CONTENT_TYPE" => "application/json",
+          :input => "{\"password\":12345}"
+        )
+      )
+    end
+
+    it "doesn't filter" do
+      initial = Kiev.config.disable_filter_for_log_levels
```
Comment (anruban) [inline spec/lib/kiev/request_body_filter_spec.rb:20]: `before` ?

File: spec/lib/kiev/request_body_filter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      Kiev.config.disable_filter_for_log_levels = [::Logger::INFO]
+      result = described_class.filter(
+        "application/json",
+        request.body,
+        ["password"],
+        []
+      )
+      expect(result).to eq("{\"password\":12345}")
+      Kiev.config.disable_filter_for_log_levels = initial
```
Comment (anruban) [inline spec/lib/kiev/request_body_filter_spec.rb:29]: `after` ?

File: spec/lib/kiev/request_body_filter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        Rack::MockRequest.env_for(
+          "/",
+          "REQUEST_METHOD" => "POST",
+          "CONTENT_TYPE" => "application/json",
+          :input => "{\"password\":12345}"
+        )
+      )
+    end
+
+    it "doesn't filter" do
```
Comment (anruban) [inline spec/lib/kiev/request_body_filter_spec.rb:19]: WDYT about adding a second context as well, when the filtering shouldn't happen?

**Review by anruban — COMMENTED** [review]
Overall look good.

File: spec/lib/kiev/request_body_filter_spec.rb (hunk 2 above)
Comment (knizhegorodov) [reply spec/lib/kiev/request_body_filter_spec.rb:20]: There is only one spec as far as I remember, so...what for? :)

File: spec/lib/kiev/request_body_filter_spec.rb (hunk 3 above)
Comment (knizhegorodov) [reply spec/lib/kiev/request_body_filter_spec.rb:29]: Same

File: spec/lib/kiev/request_body_filter_spec.rb (hunk 4 above)
Comment (knizhegorodov) [reply spec/lib/kiev/request_body_filter_spec.rb:19]: Added default case

File: spec/lib/kiev/request_body_filter_spec.rb (hunk 3 above)
Comment (everlast240) [reply spec/lib/kiev/request_body_filter_spec.rb:29]: I was thinking the same (around hook) but, yes - up to you @knizhegorodov

**Review by anruban — APPROVED** [review]

File: spec/lib/kiev/request_body_filter_spec.rb (hunk 4 above)
Comment (everlast240) [reply spec/lib/kiev/request_body_filter_spec.rb:19]: Aha, so we do have a default filter spec. Ok then

**Pull request merged**